serverless/http-on-vsock-client/src/gw_vsock_client.rs

```diff
@@ -0,0 +1,92 @@
+use clap::Parser;
+use http_on_vsock_client::vsock_client::vsock_connector;
+use hyper::{body::Body, Request, Uri};
+use serde_json::json;
+
+#[derive(Parser)]
+#[clap(author, version, about, long_about = None)]
+struct Cli {
+    /// url to query
+    #[clap(short, long, value_parser)]
+    url: String,
```

Comment:
hostname or domain might be a clearer name?

Reply:
argument looks like a url only `vsock://<cid>:<port>/`